doc/md_src_files/050_decision-points_1.md

```diff
@@ -17,10 +17,30 @@ The intent of this measure is the present state of exploitation of the vulnerabi
 | PoC <br /> (Proof of Concept) | One of the following cases is true: (1) exploit code sold or traded on underground or restricted fora; (2) typical public PoC in places such as Metasploit or ExploitDB; or (3) the vulnerability has a well-known method of exploitation. Some examples of condition (3) are open-source web proxies serve as the PoC code for how to exploit any vulnerability in the vein of improper validation of TLS certificates. As another example, Wireshark serves as a PoC for packet replay attacks on ethernet or WiFi networks. |
 | Active | Shared, observable, reliable evidence that the exploit is being used in the wild by real attackers; there is credible public reporting. |
 
+
+### Gathering Information About Exploitation
+[@householder2020historical] presents a method for searching the GitHub repositories of open-source exploit databases.
+This method could be leveraged to provide some information about whether [PoC](#exploitation) is true.
+However, part (3) of [PoC](#exploitation) would not be represented in such a search, so more information gathering would be needed.
+For part (3), perhaps we could construct a mapping of CWE-IDs which always represent vulnerabilities with well-known methods of exploitation.
+For example, CWE-295, [Improper Certificate Validation
+](https://cwe.mitre.org/data/definitions/295.html), and its child CWEs, describe improper validation of TLS certificates.
+These CWE-IDs could always be marked as [PoC](#exploitation) since that meets condition (3) in the definition.
+A comprehensive set of suggested CWE-IDs for this purpose is future work.
+
+Gathering information for [active](#exploitation) is a bit harder.
```

Comment:
I don't disagree with any of the points made here. 

Do you think it's worth saying anything about the idea that direct detection of exploitation is hard and that a lot of the "evidence of exploitation" comes from a path of incident -> malware -> re -> which vuls are involved, which really implies that the exploit has been found in deployed malware. I'm not close enough to 2020 era incident forensics to say more about the details there. But I suppose the main gist is that the path between incidents you know about and knowing which vul(s) is(were) involved is often quite lossy -- and that often the best you can do is rely on publicly available reporting.

Reply:
added to the end of the paragraph starting line 31.